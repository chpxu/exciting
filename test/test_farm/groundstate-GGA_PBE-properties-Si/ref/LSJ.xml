<?xml version="1.0" encoding="UTF-8"?>
<LSJ>
  <species n="1" chemicalSymbol="">
    <atom n="1">
      <L>-9.686997160312e-19 -6.659931767362e-20 3.101430988706e-18</L>
      <S>0.000000000000e0 0.000000000000e0 0.000000000000e0</S>
      <J>-9.686997160312e-19 -6.659931767362e-20 3.101430988706e-18</J>
    </atom>
    <atom n="2">
      <L>-9.686997160312e-19 -6.659931767362e-20 3.141665053701e-18</L>
      <S>0.000000000000e0 0.000000000000e0 0.000000000000e0</S>
      <J>-9.686997160312e-19 -6.659931767362e-20 3.141665053701e-18</J>
    </atom>
  </species>
</LSJ>
