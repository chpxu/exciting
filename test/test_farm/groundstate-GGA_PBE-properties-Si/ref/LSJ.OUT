 
Expectation values are computed only over the muffin-tin
 
Species :    1 ()
 atom :    1
  L :  -0.9686997160E-18 -0.6659931767E-19  0.3101430989E-17
  S :    0.000000000       0.000000000       0.000000000    
  J :  -0.9686997160E-18 -0.6659931767E-19  0.3101430989E-17
 atom :    2
  L :  -0.9686997160E-18 -0.6659931767E-19  0.3141665054E-17
  S :    0.000000000       0.000000000       0.000000000    
  J :  -0.9686997160E-18 -0.6659931767E-19  0.3141665054E-17
